zbark/example_test.go

```diff
@@ -39,3 +44,18 @@ func Example() {
 	// Output:
 	// {"level":"info","msg":"Bark accepts arbitrary types.","errors":0}
 }
+
+func ExampleDebarkify() {
+	logrusLogger := logrus.New()
+	logrusLogger.Out = os.Stdout
+	logrusLogger.Formatter = new(logrus.JSONFormatter)
+	logrusLogger.Formatter.(*logrus.JSONFormatter).TimestampFormat = "lies"
+	barkLogger := bark.NewLoggerFromLogrus(logrusLogger).WithField("errors", 0)
```

Comment:
`errors` comes from here and i was just matching the field use above in `ExampleBarkify()`